lib/pry/indent.rb

```diff
@@ -109,7 +109,7 @@ def initialize(pry_instance = Pry.new)
     # reset internal state
     def reset
       @stack = []
-      @indent_level = ''
+      @indent_level = ''.dup
```

Comment:
Nice work finding and fixing this! I believe we can use `String.new` to create a mutable string.

Reply:
@kyrylo thanks for your feedback! Updated the part according to your advice.

For the initial value, I added an empty string '' to ensure that the encoding stays as UTF-8 (not ASCII-8bit).
(this doesn't make a big difference in the current code base, so I'm happy to modify it as `String.new` if that's your preference.)

Reply:
Thanks for changing. I think we should either change this to just `String.new` or add a test that verifies the encoding of the string is `UTF-8`. It looks rather confusing as of now. My preference is the first option since like you mentioned it doesn't make any difference.